tests/test_crawler_container.py

```diff
@@ -96,7 +97,9 @@ def testCrawlContainer1(self):
 		'partition_strategy': {
 		    'args': {
 			'process_id': 0,
-			'num_processes': 1
+			'num_processes': 1,
+			'eventQ':multiprocessing.JoinableQueue(),
```

Comment:
yes, agreed. 

we need to think, if we can make it as a plugin and operate as a 'mode' between polling vs events. instead of replacing one another